lib/gha/set-job-vars

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+# set more GITHUB_ vars
+#
+# environment variable reference:
+#     https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/store-information-in-variables#default-environment-variables
+###
+
+set -ueo pipefail
+exec 2>&1
+
+
+gha_get_jobs() {
+  gh api "repos/$GITHUB_REPOSITORY/actions/runs/$GITHUB_RUN_ID/attempts/$GITHUB_RUN_ATTEMPT/jobs"
```

Comment:
Is the main reason for using bash so you can use the gh cli? I'd prefer if we stuck to using python for the scripts. Could we meet in the middle and utilize subprocess.run to still make use of the cli?

Reply:
Definitely.